build.js

```diff
@@ -27,24 +30,59 @@ await build({
     {
       name: 'alias',
       setup({ onResolve, resolve }) {
-        onResolve({ filter: /^fs$/ }, () => ({
-          path: fileURLToPath(new URL('src/stubs/fs.cjs', import.meta.url)),
-          sideEffects: false,
-        }));
-        onResolve({ filter: /^util$/ }, () => ({
-          path: fileURLToPath(new URL('src/stubs/util.cjs', import.meta.url)),
-          sideEffects: false,
-        }));
-        onResolve({ filter: /^vscode-emmet-helper-bundled$/ }, () => ({
-          path: fileURLToPath(new URL('src/stubs/noop.cjs', import.meta.url)),
+        // These packages are imported, but can be stubbed.
+        onResolve(
+          {
+            filter:
+              /^(detect-indent|chalk|fs|path|util|util-deprecate|vscode-emmet-helper-bundled)$/,
+          },
+          ({ path }) => ({
+            path: fileURLToPath(new URL(`src/stubs/${path}.ts`, import.meta.url)),
+            sideEffects: false,
+          }),
+        );
+
+        // The tailwindcss main export exports CJS, but we can get better tree shaking if we import
+        // from the ESM src directoy instead.
+        onResolve({ filter: /^tailwindcss$/ }, ({ path, ...options }) =>
+          resolve('tailwindcss/src', options),
+        );
+        onResolve({ filter: /^tailwindcss($|\/lib)/ }, ({ path, ...options }) =>
+          resolve(path.replace('lib', 'src'), options),
```

Comment:
that looks dangerous, but `path.replace` only replaces the first occurrence so okay i assume ^^

Reply:
This should be fine, but we should be careful doing dependency management.

Reply:
and it is fine too ^^ i was just at first a bit curious what else it might be doing ^^